packages/@expo/cli/e2e/__tests__/start-test.ts

```diff
@@ -82,7 +82,11 @@ it('runs `npx expo start --help`', async () => {
         --scheme <scheme>                      Custom URI protocol to use when launching an app
         -p, --port <port>                      Port to start the dev server on (does not apply to web or tunnel). Default: 8081
         
-        --dev-client                           Experimental: Starts the bundler for use with the expo-development-client
+        --app-launch-mode <mode>               Override the app launch mode. If not specified, the app will try to determine the best launch mode based on the current project dependencies and workflow.
+                                               start: Directly start the app
+                                               open-deep-link-dev-client: Launch expo-development-client with deep link
+                                               open-deep-link-expo-go: Launch Expo Go with deep link
+                                               open-redirect-page: Open the redirect page in browser
```

Comment:
Maybe:
```
--launch start
         dev-client
         expo-go
         browser | with-browser
```
?

- start: the application process is started. The name "direct" could also work but we want to communicate that a process on the device is launched/started/run.
- dev-client and expo-go: I removed "deep link" from both of these names since that is an implementation detail. It is useful to know that detail, though, and we could include it in the help message that explains how the dev client and Expo Go are launched.
- browser or with-browser: while the browser is more of an implementation detail than the fact there is a redirect page, I think it is more intuitive to developers to tell them that this launch mode will use a browser.